### DISCUSSION TITLE
`yaml` support

### DISCUSSION THREAD

**Discussion by scarf005** [rome/tools#3862] [Suggestions]
#1081 was closed but not because  `rome.yaml` was supported, only `rome.toml` is supported along with `rome.json`. it'd be a good idea to support `rome.yaml` natively, since yaml
- allows comments
- is easier to read for some

I'm not sure if anchor feature would be useful, but anyways YAML support would be a nice addition.

**Comment by jeysal** [comment]
I believe YAML support is a good idea once Rome also supports it natively for formatting. It's a superset of JSON, so it's just a generalization of existing JSON config.

**Reply by kubijo** [reply]
OP & the linked discussion mentions that toml config is supported, but the latest nightly doesn't seem to get `rome.toml` at all… did you drop it along the way?

**Reply by Conaclos** [reply]
It was supported in _romejs_. The rewrite in Rust led to some simplifications, including the removal of the support of `rome.toml`.

**Reply by kubijo** [reply]
And would you be able to say if additional formats support is on the horizon?

I hate TOML with passion, but it's still better than JSON what with the better ability to section-off the parts & to add comments… also it is a native citizen in rust-land, so I'd guess that the parsing should come pretty much free…

**Reply by Conaclos** [reply]
I don't think so. The development of Rome has slowed down. Supporting another format for configuration is not a priority. _JSON_ is not perfect, but it gets the job done.

**Reply by kubijo** [reply]
I see ... and I do see your point.

Not to drag this discussion needlessly on, but is there some more context for the development slowdown?

On one hand, it doesn't sound very encouraging for something I have just started to explore as replacement for eslint/prettier (those being a pretty core things for day-to-day work) and I'd like to know if there is anything I could do (wink wink, core-js).

**Reply by Conaclos** [reply]
Rome Tools Inc is no longer founded since the beginning of January. Several developers were paid to work daily on the project. Some of them are still working on the project on their spare time. New open-source contributors (including me) joined the team. This is not enough to compensate for the loss of several full-time developers.

The formatter and the linter are already well established. We can expect to fix more issues and add new lint rules in the coming months.

**Reply by kubijo** [reply]
I see ... Times are apparently pretty tight all around.

That is not great considering the great traction and good will this tooling seemingly got around some issues (eg CSS support) and what was achieved si far.

A quick look around the official page doesn't seem to yield any patronage hints though.

**Reply by ematipico** [reply]
I would consider TOML support very low priority for the time being. A nice to have. TOML is not used in the web ecosystem...

YAML though is more used (GitHub actions, CloudFormation, etc.). Between the two, YAML has objectively more usage in the web sphere. Still, I would also consider YAML a nice to have.
